charts/sysdig-deploy/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.1.7
+
+### Bugfixes
+
+* Bumped node-analyzer to 1.5.10
+* Bumped kspm-collector to 0.5.9
```

Comment:
Shouldn't this be `0.1.5` as per the [Chart.yaml bump](https://github.com/sysdiglabs/charts/pull/539/files#diff-f655c9de8ab8071079b19bc9d8717dbc40206918a3acd44ecc57349e5c2da5ddR5) above?

Reply:
Yes; updated @achandras